src/cipher.rs

```diff
@@ -943,16 +909,16 @@ impl rustls_web_pki_client_cert_verifier_builder {
     }
 }
 
-/// A server certificate verifier being constructed. A builder can be modified by,
-/// e.g. `rustls_web_pki_server_cert_verifier_builder_add_crl`. Once you're
-/// done configuring settings, call `rustls_web_pki_server_cert_verifier_builder_build`
-/// to turn it into a `rustls_server_cert_verifier`. This object is not safe
-/// for concurrent mutation.
-///
-/// See <https://docs.rs/rustls/latest/rustls/client/struct.ServerCertVerifierBuilder.html>
-/// for more information.
-pub struct rustls_web_pki_server_cert_verifier_builder {
-    _private: [u8; 0],
+box_castable! {
+    /// A server certificate verifier being constructed. A builder can be modified by,
+    /// e.g. `rustls_web_pki_server_cert_verifier_builder_add_crl`. Once you're
+    /// done configuring settings, call `rustls_web_pki_server_cert_verifier_builder_build`
+    /// to turn it into a `rustls_server_cert_verifier`. This object is not safe
+    /// for concurrent mutation.
+    ///
+    /// See <https://docs.rs/rustls/latest/rustls/client/struct.ServerCertVerifierBuilder.html>
+    /// for more information.
+    pub(crate) struct rustls_web_pki_server_cert_verifier_builder(Option<ServerCertVerifierBuilder>);
```

Comment:
I think this wasn't meant to become `pub(crate)` since it's part of hte public API.

Reply:
Good catch! Thank you. Restored to `pub`.